### PR TITLE
feat: cdn and bump cosmos client core to v0.46.0-rc7

### PR DESCRIPTION
**Pull request by YasunoriMATSUOKA**
Todo

- [x] eslint and prettier related settings
- [x] debug a file build for CDN on local environment
- [x] registered nftmodule related codec do not exist

### CODE REVIEW

**Comment by YasunoriMATSUOKA** [comment]
@mkXultra cc: @KimuraYu45z 
I have a question.
Now, I use ununifi-alpha-test-v2.cauchye.net node to debug with node-id = ununifi-alpha-test-v2.
I got the following result with https://github.com/cosmos-client/ununifi-ts/blob/feature/cdn-and-bump-cosmos-client-core-to-v46rc5/index.html

```json
{
  "tx_response": {
    "height": "0",
    "txhash": "089E49A0026CFB2F5C0DD5052A3CBDDE4C064252953B67F263632348A077DC47",
    "codespace": "sdk",
    "code": 2,
    "data": "",
    "raw_log": "unable to resolve type URL : tx parse error",
    "logs": [],
    "info": "",
    "gas_wanted": "0",
    "gas_used": "0",
    "tx": null,
    "timestamp": "",
    "events": []
  }
}
```

Is there any issue in https://github.com/cosmos-client/ununifi-ts/blob/feature/cdn-and-bump-cosmos-client-core-to-v46rc5/index.html or am I using wrong server?
If there are any issues in https://github.com/cosmos-client/ununifi-ts/blob/feature/cdn-and-bump-cosmos-client-core-to-v46rc5/index.html , plz tell me 🙏
Or, If the server was wrong, so plz tell me appropriate node server or UnUniFi/chain version 🙏

**Comment by KimuraYu45z** [comment]
It is clearly a problem related with codec. Type URL is that mean literally.
Some modules newly serve `v1` types whereas there is `v1beta`, right?
I'm suspicious about around there.

**Comment by YasunoriMATSUOKA** [comment]
I guess ununifi related tx does not use v1 or v1beta1

**Comment by KimuraYu45z** [comment]
which msg did you announced to get this error message?
Is the registration into codec of that Msg type really correct? It is my suspicion

**Comment by YasunoriMATSUOKA** [comment]
https://github.com/cosmos-client/ununifi-ts/blob/feature/cdn-and-bump-cosmos-client-core-to-v46rc5/index.html#L110-L121

**Comment by KimuraYu45z** [comment]
https://github.com/cosmos-client/ununifi-ts/blob/feature/cdn-and-bump-cosmos-client-core-to-v46rc5/src/rest/ununifi/nftmarket/index.ts#L7
weird

**Comment by KimuraYu45z** [comment]
This might be a serverside problem

File: src/cdn.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import * as ununificlient from '.';
```
Comment (KimuraYu45z) [inline src/cdn.ts:1]: Anyway, ununifi client would be better to be "default-export" like cosmosclient.
Not high priority

Comment (YasunoriMATSUOKA) [reply src/cdn.ts]: fixed

**Comment by YasunoriMATSUOKA** [comment]
I have added https://github.com/cosmos-client/ununifi-ts/pull/5/commits/3f4e8ec24e63f27033e8cf13524d28bde26e85b3 for debug only purpose.

Then, I got the following error message.

```
Uncaught (in promise) Error: This type is not registered: undefined
    at Object.b [as instanceToProtoAny] (index.min.js:2:611061)
    at index.html:124:40
```

It means frontend issue?

**Comment by KimuraYu45z** [comment]
Yes it means the code for registration into codec is not executed.

**Comment by YasunoriMATSUOKA** [comment]
with codec register, tx succeeded.

```
{
  "tx_response": {
    "height": "241175",
    "txhash": "D533BB71782DE5C825DA624349D92F8710D9DA25A297BBC70720DDA6243C19C2",
    "codespace": "",
    "code": 0,
    "data": "12270A252F756E756E6966692E6E66746D61726B65742E4D73674C6973744E6674526573706F6E7365",
    "raw_log": "[{\"msg_index\":0,\"events\":[{\"type\":\"message\",\"attributes\":[{\"key\":\"action\",\"value\":\"list_nft\"}]},{\"type\":\"ununifi.nftmarket.EventListNft\",\"attributes\":[{\"key\":\"owner\",\"value\":\"\\\"ununifi18ytx2jmfp00unst6hn3hlg6vfz0praqc88q003\\\"\"},{\"key\":\"class_id\",\"value\":\"\\\"a10\\\"\"},{\"key\":\"nft_id\",\"value\":\"\\\"a10\\\"\"}]}]}]",
    "logs": [
      {
        "msg_index": 0,
        "log": "",
        "events": [
          {
            "type": "message",
            "attributes": [
              {
                "key": "action",
                "value": "list_nft"
              }
            ]
          },
          {
            "type": "ununifi.nftmarket.EventListNft",
            "attributes": [
              {
                "key": "owner",
                "value": "\"ununifi18ytx2jmfp00unst6hn3hlg6vfz0praqc88q003\""
              },
              {
                "key": "class_id",
                "value": "\"a10\""
              },
              {
                "key": "nft_id",
                "value": "\"a10\""
              }
            ]
          }
        ]
      }
    ],
    "info": "",
    "gas_wanted": "0",
    "gas_used": "0",
    "tx": null,
    "timestamp": "",
    "events": [
      {
        "type": "tx",
        "attributes": [
          {
            "key": "fee",
            "value": "",
            "index": true
          }
        ]
      },
      {
        "type": "tx",
        "attributes": [
          {
            "key": "acc_seq",
            "value": "ununifi18ytx2jmfp00unst6hn3hlg6vfz0praqc88q003/47",
            "index": true
          }
        ]
      },
      {
        "type": "tx",
        "attributes": [
          {
            "key": "signature",
            "value": "ue6tuN8Dm67yCz4Cz/Vqncxs6W9gFnF7JRWPGSpS1fEQi7u3XtuoMokgtmZutGCosM+GrXF6k8mWsMzp3QY8CA==",
            "index": true
          }
        ]
      },
      {
        "type": "message",
        "attributes": [
          {
            "key": "action",
            "value": "list_nft",
            "index": true
          }
        ]
      },
      {
        "type": "ununifi.nftmarket.EventListNft",
        "attributes": [
          {
            "key": "owner",
            "value": "\"ununifi18ytx2jmfp00unst6hn3hlg6vfz0praqc88q003\"",
            "index": true
          },
          {
            "key": "class_id",
            "value": "\"a10\"",
            "index": true
          },
          {
            "key": "nft_id",
            "value": "\"a10\"",
            "index": true
          }
        ]
      }
    ]
  }
}
```

but I don't know how to resolve the fundamental issue registered codec do not exist.

**Comment by KimuraYu45z** [comment]
for latest commit, it is just a wrong usage.

**Comment by YasunoriMATSUOKA** [comment]
@KimuraYu45z 
plz review 🙏

File: tsconfig.json [hunk 2]
```diff
@@ -60,8 +60,9 @@
     // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
     /* Advanced Options */
-    "skipLibCheck": true, /* Skip type checking of declaration files. */
-    "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
+    "skipLibCheck": true /* Skip type checking of declaration files. */,
+    "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */,
+    "downlevelIteration": true,
```
Comment (KimuraYu45z) [inline tsconfig.json:65]: I said please delete this line

**Review by KimuraYu45z — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 2 above)
Comment (YasunoriMATSUOKA) [reply tsconfig.json]: fixed

**Review by KimuraYu45z — APPROVED** [review]
LGTM

**Pull request merged**